src/App.css

```diff
@@ -0,0 +1,63 @@
+@font-face {
+  font-family: 'Beleren';
+  src: url('/Beleren-Bold.ttf') format('truetype');
+  font-weight: bold;
+  font-style: normal;
+}
+
+body {
+  background: url('Gray-Patterned-BG.jpg');
+  color: whitesmoke;
+  font-family: 'Beleren', sans-serif;
+  font-weight: bold;
+  margin: 0rem;
+}
+
+header {
+  border-bottom: 1px solid grey;
+  font-size: 3rem;
+  padding: .5rem 0rem .5rem 0rem;
+  text-align: center;
+}
+
+.card {
+  border: 1px solid grey;
+  border-radius: 1.5rem;
+  padding: .5rem;
+  margin: 1rem;
+  width: 25rem;
+}
+
+.life-counter {
+  background-color: #111111;
+  border: .25rem solid #444444;
```

Comment:
Might want to set these colors as CSS variables for future use on different elements.